pyairtable/api/api.py

```diff
@@ -17,17 +18,24 @@ class Api(ApiAbstract):
         >>> api.all('base_id', 'table_name')
     """
 
-    def __init__(self, api_key: str, timeout=None):
+    def __init__(
+        self,
+        api_key: str,
+        timeout=None,
+        max_records_per_request=MAX_RECORDS_PER_REQUEST,
```

Comment:
WDYT about `max_records_per_request= ApiAbstract.DEFAULT_MAX_RECORDS_PER_REQUEST` instead of importing the constant?

Reply:
Hmm it's a module variable, so it can't be imported like that

Reply:
I think it can! As long as you do `import ApiAbstract`

Reply:
It's a module level variable, not a class variable, so it could be abstract.DEFAULT_MAX_RECORDS_PER_REQUEST I guess